### ISSUE TITLE
Merging cell's border styles

### ISSUE THREAD

**Issue by jean-marc-voillequin** [axlsx-styler-gem/axlsx_styler#31]
Hello,

I want:
![expected](https://user-images.githubusercontent.com/56127093/112454288-d11e6280-8d58-11eb-9ea7-e65ed4f11909.png)

My code is:
```
require 'axlsx_styler'
axlsx = Axlsx::Package.new
workbook = axlsx.workbook
workbook.add_worksheet do |sheet|
	sheet.add_row
	row = sheet.add_row
	row.add_cell
	cell = row.add_cell
	sheet.add_border('B2:B2', { edges: [:left, :right], style: :thick, color: 'FF0000' })
	sheet.add_border('B2:B2', { edges: [:top, :bottom], style: :double, color: '0000FF' })
end
axlsx.serialize('a.xlsx')

```

But i get:
![found](https://user-images.githubusercontent.com/56127093/112454356-e3989c00-8d58-11eb-90c6-d00744fe868b.PNG)

Any idea to perform that?
Thanks!

**Comment by westonganger** [comment]
Its not immediately obvious to me whats causing this. 

Initial call is https://github.com/axlsx-styler-gem/axlsx_styler/blob/eb13ce7f9cd2a2419e8e15675bbd055e388d65cb/lib/axlsx_styler/axlsx_worksheet.rb#L39-L48

Which calls `BorderCreator` which is a small class easily understandable https://github.com/axlsx-styler-gem/axlsx_styler/blob/master/lib/axlsx_styler/border_creator.rb

Please try and fiddle with the coden make some changes to see if you can get something working.

**Comment by westonganger** [comment]
This issue is now resolved as a [PR](https://github.com/caxlsx/caxlsx/pull/117) has been created, merged and released in caxlsx@3.2.0
